cluster/tool/cluster-cli/main.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
this could go in a cmd package, to avoid the long cluster/tool/auth-cli package, IMO